### DISCUSSION TITLE
Language specific minimum string length before firing blink?

### DISCUSSION THREAD

**Discussion by benfrain** [saghen/blink.cmp#495] [Q&A] [answered]
I write a lot in Neovim. One thing that would be great is if it was possible to tell, say markdown files, to only fire Blink if the string is over X characters.

Is this achievable with blink currently?

**Comment by saghen** [comment, marked as answer]
Opened an issue for this #502. You could get around this temporarily by using `min_keyword_length = function() vim.bo.filetype == 'markdown' and X or 0 end` on all of your providers

**Reply by benfrain** [reply]
Lovely, thanks
